docs/docs/writing_your_plugin.md

```diff
@@ -192,7 +192,7 @@ To add your own functionality via the plugin interface:
       [`sb->schedule(...)`][14].
 			
     - If you spin your own threads, they **must** wait for
-          `pb->lookup_impl<Stoplight>()->wait_for_ready()` the first time they
+          `pb->lookup_impl<stoplight>()->wait_for_ready()` the first time they
```

Comment:
Is this supposed to be lowercase? From the implementation of [`lookup_impl()`](https://github.com/ILLIXR/ILLIXR/blob/1963c106d4850a2a5749584f265592fb8c9bd899/include/illixr/phonebook.hpp#L122), the call would indeed look like `pb->lookup_impl<Stoplight>()->wait_for_ready()`.

The offloading VIO plugin that uses `lookup_impl` has [`pb->lookup_impl<Stoplight>()`](https://github.com/ILLIXR/ILLIXR/blob/1963c106d4850a2a5749584f265592fb8c9bd899/plugins/offload_vio/device_tx/plugin.cpp#L36C24-L36C52).